### ISSUE TITLE
Multi-Language Support for Frontend

### ISSUE THREAD

**Issue by yashpandey06** [OWASP/Nest#224]
**Is your feature request related to a problem? Please describe.**  
Currently, the website supports only a single language, which limits its accessibility and usability for non-English speaking users. 

**Describe the solution you'd like**  
I would like to implement multi-language support on the client side. Key features of the solution include:  
1. A language selector (dropdown or toggle) for users to choose their preferred language.  
2. Localization of all user-facing text using a library like `i18next`.  
3. Language persistence using local storage or cookies to save the user’s preference.  
4. A fallback language (English) in case a translation is unavailable.  

Please share your thoughts or suggestions for additional features that can improve this implementation.

**Comment by yashpandey06** [comment]
FYI  ( @everyone ): As per discussion on slack , this is the not the priority issue right now ...but we will surely look into it in upcoming iterations.

**Comment by yashpandey06** [comment]
@arkid15r do we have any upcoming plan for these issues ?

**Comment by arkid15r** [comment]
No, not yet, it's still a backlog P3 task (you can see the details on the right)

**Comment by SEEDART007** [comment]
@yashpandey06 hey can I add up this feature? you need how many language supported in it?

**Comment by arkid15r** [comment]
@SEEDART007 this is a backlog one, we didn't prioritize it so not action required yet.

**Comment by Ani07-05** [comment]
@arkid15r  could I work on this feature
